src/include/daos/object.h

```diff
@@ -437,7 +439,7 @@ struct daos_recx_ep_list {
 	/** #total items (capacity) in re_items array */
 	uint32_t		 re_total;
 	/** epoch valid flag, re_items' re_ep can be ignored when it is false */
-	uint32_t		 re_ep_valid:1;
+	bool			 re_ep_valid;
```

Comment:
Just curious, why change this from 1 bit to 32 bit?

Reply:
since there is only one bit now, seems bool is more clear?